hibernate-core/src/test/java/org/hibernate/orm/test/bytecode/enhancement/lazy/proxy/LazyToOnesProxyWithSubclassesTest.java

```diff
@@ -281,8 +281,6 @@ public void cleanUpTestData() {
 		inTransaction(
 				session -> {
 					session.createQuery( "delete from OtherEntity" ).executeUpdate();
-					session.createQuery( "delete from Human" ).executeUpdate();
-					session.createQuery( "delete from Primate" ).executeUpdate();
```

Comment:
What is this for?

Reply:
I have an annoying failure when I run the tests locally, the issue with temporary tables during deletion, I made a tried and forget to revert this change for the PR

Reply:
I found the issue and fixed it as part of https://github.com/hibernate/hibernate-orm/pull/4425